### ISSUE TITLE
Live stream requests MPD over and over and doesn't start playing immediately.

### ISSUE THREAD

**Issue by adgllorente** [google/shaka-player#1841]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
2.4.7

**Can you reproduce the issue with our latest release version?**
No

**Can you reproduce the issue with the latest code from `master`?**
No

**Are you using the demo app or your own custom app?**
Demo app with custom header

**If custom app, can you reproduce the issue using our demo app?**

**What browser and OS are you using?**
Chrome 72.0.3626.121

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**

**What are the manifest and license server URIs?**
We are using a stream with this information:

- MPD: https://b41557.cdn.telefonica.com/live/disk1/NatGeoHD/NATIONAL_GEOGRAPHIC_CN_DASH_HD/NatGeoHD.mpd
- Issuer: https://wvcypherdrm.gvp.telefonica.com/DashIssuer.aspx
- Custom header:

```
  player.getNetworkingEngine().registerRequestFilter(function (type, request) {
    if (type == shaka.net.NetworkingEngine.RequestType.LICENSE) {
      request.headers = {
        'x-authentication': 'AU_ZGAE6U_t_9qkWlVsonRVIhMgjB1MoIADmMw5Q2Uozy53gCgf9YhblLgPgY13a7ejMjlTgQHPP5jDiXzetyWHj8-ny6c0L5F5HQXl14aCWiBUKQKZEZFq6YB2WOf8cYP1hzEzhjp7tSK9MNxgcxtIQhtwsQP1eVWqmHFSDFAZeVKWhAP1ak_5oka_bsvCiZRmpz2UVrh2tzwAhd35yVNx-eUpoFVHUym-MzaIU9-xkfk8PnXNVc'
      };
    }
  });
```

**What did you do?**
Just click on play.

**What did you expect to happen?**
Live stream plays correctly.

**What actually happened?**
I've seen MPD is requested several times, playback doesn't start immediately and it takes almost 25-30 seconds for the Stream to start.

Logs are reporting many messages like these:

```
Assertion failed: SegmentReferences are incorrect
(audio:1) cannot find segment: currentPeriod.startTime=0 lookupTime=132407834.00999999
```

This is not happening in 2.5.0 so I've been reviewing existing issues and I've found this issue that may be related: https://github.com/google/shaka-player/issues/1727.

I would be pleased if you tell me why is this happening or if there's a problem with the stream to report all the information to the team generating the stream.

Thanks!

**Comment by vaage** [comment]
@adgllorente thank you for filing an issue with us. I am trying to reproduce your issue, but I am getting a 403 (forbidden) error when requesting the manifest. How can we access your manifest?

**Comment by adgllorente** [comment]
Hi @vaage, we've been reviewing the MPD with our Video Core Team and we have found that the MPD has a clock sync issue on the availabilityStartTime property.

We've seen in the issue attached (#1727) that new shaka release does not depend on the availabilityStartTime and uses the internal time from the fragments instead.

So that, we have removed the clock sync issue from the stream and playback is working fine in both 2.4 and 2.5 releases of the shaka-player.

I'm closing the issue because no more support is needed from you, thank you so much for your time.
